src/ctia/store.clj

```diff
@@ -27,16 +25,19 @@
   (delete-identity [this org-id role]))
 
 (defprotocol IEventStore
-  (read-event [this id ident params])
-  (create-events [this new-events])
-  (list-events [this filtermap ident params]))
+  (create-events [this new-events]))
```

Comment:
`create-record` accepts multiple records. I'm wondering if we could get rid of the `IEventStore`. Not in the scope of this PR

Reply:
The only reason I kept IEventStore protocol is that the signatures of create-events and create-records are incompatible. This could be solved by adding an arity-2 variant for create-records of IStore protocol. But as you mention, it would be good to keep it in separate PR.